k8s/nginx/staging/global-config.yaml

```diff
@@ -41,17 +48,25 @@ data:
       set $method POST;
       set $verification_path verify?tenant=airqo;
 
-      if ($request_uri ~* 'token=(?<token>[^&]+)') {
+      if ($arg_token){
```

Comment:
Hi @Psalmz777 , many thanks for these welcome changes. Please shed more light on these changes, what could we be aiming to achieve here?